printer_mods/DoubleT/input_shaper_mount/readme.md

```diff
@@ -0,0 +1,14 @@
+# Input shaper mount for Afterburner
+
+This mode replaces the front middle skirt in order to mount a 3.5in touchscreen display for Klipperscreen. It has two printed parts: the main body and the front plate. Threaded inserts go into the main body and the M3x8 screws are used to mount the front plate onto the main body.
+
+![image1](./images/image1.png)
+
```

Comment:
Where does this mount on the Afterburner and what are the magnets and heatset inserts for?